examples/hybrid/cli_options.jl

```diff
@@ -310,9 +310,9 @@ function parse_commandline()
         help = "Check conservation of mass and energy [`false` (default), `true`]"
         arg_type = Bool
         default = false
-        "--non_orographic_gravity_wave"
-        help = "Apply parameterization for convective gravity wave forcing on horizontal mean flow"
-        arg_type = Bool
+        "--gravity_wave"
+        help = "Apply parameterization for convective gravity wave forcing on horizontal mean flow [`false` (default), `NonOrographic`]"
+        arg_type = String
```

Comment:
I prefer to keep the `--non_orographic_gravity_wave` option as Boolean and add a `--orographic_drag` option as Bool later.

Reply:
I've reverted it back to `non_orographic_gravity_wave`, which is still a Bool, and that is later converted to a `NonOrographyGravityWave` or `Nothing` type, depending on the Bool. This will help improve inference in our tendency function.